content/spin/v2/cli-reference.md

```diff
@@ -84,4244 +90,881 @@ OPTIONS:
-    -v, --value <VALUES>               Parameter values to be passed to the template (in name=value
-                                       format)
-        --values-file <VALUES_FILE>    A TOML file which contains parameter values in name = "value"
-                                       format. Parameters passed as CLI option overwrite parameters
-                                       specified in the file
-```
+{{ blockEnd }}
+
+<!-- markdownlint-disable-next-line titlecase-rule -->
+## spin cloud sqlite
+
+{{ tabs "spin-version" }}
+
+{{ startTab "v2.0"}}
+
+The `spin cloud sqlite` command is implemented by the [Fermyon Cloud Plugin](../cloud/cloud-command-reference#spin-cloud-sqlite).
 
 {{ blockEnd }}
 
-{{ startTab "v1.5"}}
+{{ blockEnd }}
 
-<!-- @selectiveCpy -->
+<!-- markdownlint-disable-next-line titlecase-rule -->
+## spin cloud unlink
 
-```console
-$ spin add --help
+{{ tabs "spin-version" }}
 
-spin-add 
-Scaffold a new component into an existing application
+{{ startTab "v2.0"}}
 
+The `spin cloud unlink` command is implemented by the [Fermyon Cloud Plugin](../cloud/cloud-command-reference#spin-cloud-unlink).
 
-USAGE:
-    spin add [OPTIONS] [ARGS]
+{{ blockEnd }}
 
-ARGS:
-    <TEMPLATE_ID>    The template from which to create the new application or component. Run
-                     `spin templates list` to see available options
-    <NAME>           The name of the new application or component
+{{ blockEnd }}
 
-OPTIONS:
-    -a, --accept-defaults              An optional argument that allows to skip prompts for the
-                                       manifest file by accepting the defaults if available on the
-                                       template
-    -f, --file <APP_MANIFEST_FILE>     Path to spin.toml
-    -h, --help                         Print help information
-    -o, --output <OUTPUT_PATH>         The directory in which to create the new application or
-                                       component. The default is the name argument
-        --tag <TAGS>                   Filter templates to select by tags
-    -v, --value <VALUES>               Parameter values to be passed to the template (in name=value
-                                       format)
-        --values-file <VALUES_FILE>    A TOML file which contains parameter values in name = "value"
-                                       format. Parameters passed as CLI option overwrite parameters
-                                       specified in the file
-```
+<!-- markdownlint-disable-next-line titlecase-rule -->
+## spin cloud variables
+
+{{ tabs "spin-version" }}
+
+{{ startTab "v2.0"}}
+
+The `spin cloud variables` command is implemented by the [Fermyon Cloud Plugin](../cloud/cloud-command-reference#spin-cloud-variables).
 
 {{ blockEnd }}
 
 {{ blockEnd }}
 
 <!-- markdownlint-disable-next-line titlecase-rule -->
-## spin build
+## spin deploy
 
 {{ tabs "spin-version" }}
 
-{{ startTab "v1.0"}}
+{{ startTab "v2.0"}}
```

Comment:
For `spin login` you just say it's a shortcut to `spin cloud login` - should we not do the same here?

Reply:
Yes, thanks @itowlson